app/build.gradle

```diff
@@ -24,14 +24,14 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation project(':msglibrary')
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.android.material:material:1.1.0'
     implementation 'com.google.android.gms:play-services-location:17.0.0'
     implementation 'androidx.room:room-runtime:2.2.3'
     implementation 'com.github.rehacktive:waspdb:1.1.1'
     implementation "androidx.fragment:fragment:1.2.1"
+    implementation 'com.github.Cogno-IDU:smslibrary:v2.2'
```

Comment:
I know we said the "common" library, but we also specified our fork of the library, otherwise where do all our random generators go?